api/v1alpha3/conversion_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	. "github.com/onsi/gomega"
 
 	fuzz "github.com/google/gofuzz"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
```suggestion
	""k8s.io/utils/pointer"
```

Only use aws.Bool when working with AWS services.